raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -523,7 +546,7 @@ contract TokenNetwork is Utils {
         channel.participants[closing_participant].is_the_closer = true;
 
         // This is the block number at which the channel can be settled.
-        channel.settle_block_number += uint256(block.number);
+        channel.settle_block_number = settlement_timeout + block.number;
```

Comment:
Probably not important for a PoC but this could overflow if the contract is deployed with a very high `settlement_timeout_min`